CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ _None_
 ### Bug Fixes
 
 * Fix `ios_lint_localizations` action so that it no longer mistakely reports missing keys not yet translated in the other locales' `.strings` as violations. [#353]
+* Fix `an_localize_libs` so that it does not move XML nodes around when merging lib strings (and replace them in-place instead). [#358]
```

Comment:
I'm not sure that was a bug. Maybe an oversight of the first implementation? Something that became obvious only after we started using it "at scale"?

Anyways, that's just me being pedantic. 🙃

Reply:
Fair; but even if the "Bug Fixes" wording might not be the best for that specific case, that is still the most appropriate SemVer section matching this kind of change, as this is not a "Breaking Change", nor really a "New Feature", but this is not just "Internal" either as it has impacts on the result.

And overall I think it still makes sense to consider this was a bug, even if not affecting the behavior of the app using the resulting `strings.xml`, but as it was affecting developers working on the file (introducing blank lines, but most importantly dissociating XML nodes from their nearby associated XML comment) and could lead to misleading understanding of the XML file by developers later due to those unexpected changes. So not a "bug" as in "the software using this wasn't  working", but still one as in "not producing the output that developers expected" 🙃